server/app/views/applicant/ProgramCardsSectionFragment.html

```diff
@@ -39,6 +39,7 @@
       <h3
         class="usa-card__heading cf-application-card-title cf-prose-h4"
         th:text="${card.title()}"
+        th:data-program-id="${card.programId()}"
```

Comment:
Adding this because I need a stable and straightforward way to check for the program id of a card in browser tests. Without it, it was having to jump through too many hoops to get what I needed.

Reply:
That makes sense and I think it will be handy for other tests as well.